backend/pom.xml

```diff
@@ -288,6 +290,18 @@
             <version>1.16.20</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.mapstruct</groupId>
+            <artifactId>mapstruct-jdk8</artifactId>
+            <version>${org.mapstruct.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.mapstruct</groupId>
+            <artifactId>mapstruct-processor</artifactId>
+            <version>${org.mapstruct.version}</version>
+            <scope>provided</scope>
```

Comment:
Why provided ? Add comment to explain

Reply:
You right this is not useful, i will delete it